examples/dark-mode/app/entry.server.tsx

```diff
@@ -0,0 +1,23 @@
+import { renderToString } from "react-dom/server";
+import { RemixServer } from "remix";
+import type { EntryContext } from "remix";
+
+require("dotenv").config();
```

Comment:
This should not be needed, none example adds dotenv to load env variables

Reply:
`dotenv` has been removed 👍